src/server.cpp

```diff
@@ -1958,6 +1958,13 @@ void Server::SendCSMRestrictionFlags(session_t peer_id)
 	Send(&pkt);
 }
 
+void Server::SendPlayerSpeed(session_t peer_id, const v3f &added_vel)
+{
+	NetworkPacket pkt(TOCLIENT_PLAYER_SPEED, 0, peer_id);
```

Comment:
i suggest to replace fast move if this pr is merged with this implementation, it permits to servers to set fast move velocity :)

Reply:
How should this replace fast move? The velocity is only added once.

Reply:
Servers already can set fast move velocity though, just not per-player.

Reply:
I can't see how this can set fast move velocity, it doesn't seem related.